packages/stream-caip10-link-handler/src/caip10-link-handler.ts

```diff
@@ -38,6 +38,10 @@ export class Caip10LinkHandler implements StreamHandler<Caip10Link> {
     context: StreamReaderWriter,
     state?: StreamState
   ): Promise<StreamState> {
+    if (process.env.CERAMIC_ENABLE_V4_MODE) {
```

Comment:
would it be worthwhile to throw errors when creating these stream types, not just when applying a commit? a lot of the tests _may_ pass because they use tile documents but don't actually load/apply a commit, but we should probably skip them and update them to use MIDs still (?)

Reply:
There's no such thing as creating a stream without applying a commit.  A stream is created by applying the genesis commit.